ad-you-like/DOCUMENTATION.md

```diff
@@ -0,0 +1,81 @@
+# Ad You Like
+## General Compatibility
+|Feature|  |
+|---|---|
+| Consent |  |
+| Native Ad Support |  |
```

Comment:
Please try and fill out these tables here.
you can reference examples of other adapters, like  
https://github.com/Adyoulike/ix-library-adapters/blob/AdYouLike-adaptor/emx-digital/DOCUMENTATION.md

Reply:
Done